databases/week2/sql_Week2Part4.sql

```diff
@@ -0,0 +1,28 @@
+CREATE DATABASE company;
+
+USE company;
+
+CREATE TABLE employees(
+id INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+first_name VARCHAR(45) NOT NULL,
```

Comment:
Be careful, names can be veeeeey long, i'd set the length to 255 (the maximum, as far as i know).

Reply:
Yes you are right. Names ca be pretty long